pkg/devfile/parser/data/common/command_helper.go

```diff
@@ -59,6 +59,16 @@ func (dc DevfileCommand) GetExecWorkingDir() string {
 	return ""
 }
 
+// IsExec checks if the command is an exec command
+func (dc DevfileCommand) IsExec() bool {
```

Comment:
New function means new tests :)

Despite this test being extremely small haha, we should add a test function for this!

Reply:
@cdrage yes, I'll add the new test in ;)

could you remove the lgtm label? it looks like it got added?